include/crocoddyl_msgs/whole_body_state_publisher.h

```diff
@@ -137,6 +137,46 @@ class WholeBodyStateRosPublisher {
     }
   }
 
+  /**
+   * @brief Publish a whole-body state ROS message
+   *
+   * @param t[in]    Time in secs
+   * @param q[in]    Configuration vector (dimension: model.nq)
+   * @param v[in]    Generalized velocity (dimension: model.nv)
+   * @param a[in]    Generalized acceleration (dimension: model.nv)
+   * @param tau[in]  Joint effort (dimension: model.nv)
+   * @param p[in]    Contact position
+   * @param pd[in]   Contact velocity
+   * @param f[in]    Contact force, type and status
+   * @param s[in]    Contact surface and friction coefficient
+   */
+  void
+  publish(const double t, const Eigen::Ref<const Eigen::VectorXd> &q,
+          const Eigen::Ref<const Eigen::VectorXd> &v,
+          const Eigen::Ref<const Eigen::VectorXd> &a,
+          const Eigen::Ref<const Eigen::VectorXd> &tau,
+          const std::map<std::string, pinocchio::SE3> &p,
+          const std::map<std::string, pinocchio::Motion> &pd,
+          const std::map<std::string, std::tuple<pinocchio::Force, ContactType,
+                                                 ContactStatus>> &f,
+          const std::map<std::string, std::pair<Eigen::Vector3d, double>> &s) {
```

Comment:
Include default values as done in previous case, i.e.,
```suggestion
          const std::map<std::string, pinocchio::SE3> &p = DEFAULT_SE3,
          const std::map<std::string, pinocchio::Motion> &pd  = DEFAULT_MOTION,
          const std::map<std::string, std::tuple<pinocchio::Force, ContactType,
                                                 ContactStatus>> &f = DEFAULT_FORCE,
          const std::map<std::string, std::pair<Eigen::Vector3d, double>> &s = DEFAULT_FRICTION) {
```